srcpkgs/hashcash/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'hashcash'
+pkgname=hashcash
+version=1.23
+revision=1
+archs="~*-musl"
+build_style=gnu-makefile
+make_build_args="hashcash docs"
```

Comment:
```
=> hashcash-1.23_1: running do_install ...
cc -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe         -c -o sha1.o sha1.c
cc sha1.o libsha1.o -o sha1 -Wl,-z,relro -Wl,-z,now -Wl,--as-needed    
```

It's building sha1 during install.

Reply:
Yes, in `Makefile` it has an option to build either with openssl or with embedded sha1 implementation.

Reply:
What I meant is that this should be moved to `do_build`. And generally speaking I'd expect openssl/libressl SHA1 to perform better.

Reply:
Sorry, I missed that. Fixed in a new commit: https://github.com/void-linux/void-packages/pull/28379/commits/bad76bf16e5b0a45bfa9ca69a124ad176a35b4f2